[Flaky] test_numpy_op.test_np_einsum

## Description
```
======================================================================

FAIL: test_numpy_op.test_np_einsum

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/usr/local/lib/python3.5/dist-packages/nose/case.py", line 198, in runTest

    self.test(*self.arg)

  File "/work/mxnet/tests/python/unittest/common.py", line 177, in test_new

    orig_test(*args, **kwargs)

  File "/work/mxnet/python/mxnet/util.py", line 315, in _with_np_shape

    return func(*args, **kwargs)

  File "/work/mxnet/python/mxnet/util.py", line 499, in _with_np_array

    return func(*args, **kwargs)

  File "/work/mxnet/tests/python/unittest/test_numpy_op.py", line 3524, in test_np_einsum

    assert_almost_equal(out_mx.asnumpy(), expected_np, rtol=rtol, atol=atol)

  File "/work/mxnet/python/mxnet/test_utils.py", line 627, in assert_almost_equal

    raise AssertionError(msg)

AssertionError: 

Items are not equal:

Error 1.669922 exceeds tolerance rtol=1.000000e+00, atol=1.000000e-01 (mismatch 25.000000%).

Location of maximum error: (0, 1), a=0.24609375, b=0.02963257

 ACTUAL: array([[156.    ,   0.2461],

       [433.2   , -30.8   ]], dtype=float16)

 DESIRED: array([[156.     ,   0.02963],

       [433.2    , -30.72   ]], dtype=float16)

-------------------- >> begin captured stdout << ---------------------


*** Maximum errors for vector of size 4:  rtol=1.0, atol=0.1


  1: Error 1.669922  Location of error: (0, 1), a=0.24609375, b=0.02963257


--------------------- >> end captured stdout << ----------------------

-------------------- >> begin captured logging << --------------------

root: INFO: NumPy-shape semantics has been activated in your code. This is required for creating and manipulating scalar and zero-size tensors, which were not supported in MXNet before, as in the official NumPy library. Please DO NOT manually deactivate this semantics while using `mxnet.numpy` and `mxnet.numpy_extension` modules.

common: INFO: Setting test np/mx/python random seeds, use MXNET_TEST_SEED=1120100586 to reproduce.

--------------------- >> end captured logging << ---------------------
```

## Occurrences
Unrelated PR - #16599 

http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Funix-cpu/detail/PR-16599/5/pipeline

## What have you tried to solve it?
Nothing

--- a/tests/python/unittest/test_numpy_op.py
+++ b/tests/python/unittest/test_numpy_op.py
@@ -3503,7 +3503,7 @@ def test_np_einsum():
         # ('abiz,abjz->abij', [(64, 8, 128, 512), (64, 8, 128, 512)], lambda *args: (_np.matmul(_np.ones((64, 8, 128, 128)), args[1]),
         #                                                                            _np.matmul(_np.ones((64, 8, 128, 128)), args[0]))),
     ]
-    dtypes = ['float16', 'float32', 'float64', 'int32']
+    dtypes = ['float32', 'float64', 'int32']
     acc_type = {'float16': 'float32', 'float32': 'float64', 'float64': 'float64',
                 'int32': 'int64'}
     for hybridize in [False, True]:

